modules/statics/src/coins.ts

```diff
@@ -7536,6 +7544,14 @@ export const coins = CoinMap.fromCoins([
     '0x8af17a6396c8f315f6b6dbc6aa686c85f9b3e554',
     UnderlyingAsset.XTZBULL
   ),
+  erc20(
+    'fec45ebe-80a0-4cfa-8fb3-7c4100517f18',
+    'xusd',
+    'XUSD',
+    6,
+    '0xc08e7e23c235073c6807c2efe7021304cb7c2815',
+    UnderlyingAsset.XUSD
+  ),
```

Comment:
Why are we not adding this ?

Reply:
We are not adding this in the base.ts because we already have this token for other coin (In this case, Solana)